sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlClient.java

```diff
@@ -0,0 +1,165 @@
+package com.azure.security.keyvault.administration;
+
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.security.keyvault.administration.implementation.models.KeyVaultErrorException;
+import com.azure.security.keyvault.administration.implementation.models.RoleAssignment;
+import com.azure.security.keyvault.administration.implementation.models.RoleAssignmentProperties;
+import com.azure.security.keyvault.administration.implementation.models.RoleDefinition;
+import com.azure.security.keyvault.administration.models.RoleScope;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+/**
+ * The {@link KeyVaultAccessControlClient} provides synchronous methods to view and manage Role Based Access for the
+ * Azure Key Vault. The client supports creating, listing, updating, and deleting {@link RoleAssignment role
+ * assignments}. Additionally, the client supports listing {@link RoleDefinition role definitions}.
+ */
+@ServiceClient(builder = KeyVaultAccessControlClientBuilder.class)
+public class KeyVaultAccessControlClient {
+    private final KeyVaultAccessControlAsyncClient asyncClient;
+
+    /**
+     * Creates an {@link KeyVaultAccessControlClient} that uses a {@link com.azure.core.http.HttpPipeline pipeline}
+     * to service requests.
+     *
+     * @param asyncClient The {@link KeyVaultAccessControlAsyncClient} that this client routes its request through.
+     */
+    KeyVaultAccessControlClient(KeyVaultAccessControlAsyncClient asyncClient) {
+        this.asyncClient = asyncClient;
+    }
+
+    /**
+     * Gets the URL for the Key Vault this client is associated with.
+     *
+     * @return The Key Vault URL.
+     */
+    public String getVaultUrl() {
+        return asyncClient.getVaultUrl();
+    }
+
+    /**
+     * Get all {@link RoleDefinition role definitions} that are applicable at the given {@link RoleScope
+     * scope} and above.
+     *
+     * @param scope   The {@link RoleScope scope} of the {@link RoleDefinition role definitions}.
+     * @param context Additional {@link Context} that is passed through the HTTP pipeline during the service call.
+     * @return A {@link PagedIterable} containing the {@link RoleDefinition role definitions} for the given
+     * {@link RoleScope scope}.
+     * @throws KeyVaultErrorException if the operation is unsuccessful.
+     * @throws NullPointerException   if the {@link RoleScope scope} is {@code null}.
+     */
+    public PagedIterable<RoleDefinition> listRoleDefinitions(RoleScope scope, Context context) {
```

Comment:
Since this scope is used for both `RoleAssignment` and `RoleDefinition` I found `RoleScope` to be more fitting, else it makes me think that it's only reserved for the former.

Reply:
Let's consider snapping to the names discussed [here](https://github.com/Azure/azure-sdk-for-js/pull/10815#discussion_r477475161).

Reply:
Agreed to the naming proposed.